src/document-check.js

```diff
@@ -17,37 +20,58 @@ const tasks = new Listr(
       /**
        * @param {GlobalOptions & DocsVerifierOptions} ctx
        */
-      enabled: ctx => hasTsconfig,
+      enabled: ctx => hasTsconfig && !isTypescript,
```

Comment:
This means it'll run on typed ESM/CJS only but not TypeScript projects?

Reply:
Good catch, thanks!